examples/asr/conf/fastconformer/fast-conformer_transducer_bpe.yaml

```diff
@@ -254,7 +254,6 @@ trainer:
   precision: 32 # 16, 32, or bf16
   log_every_n_steps: 10  # Interval of logging.
   enable_progress_bar: True
-  resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.
```

Comment:
Were these configs not updated in the PTL 2 update ?

Reply:
No, actually all other asr and even SV and SD configs need to be updated too.

Reply:
This is odd - why were they skipped and how did tests pass then? @athitten

Reply:
@titu1994 during the upgrade we just removed it for the yaml files that complained of this error. If you still see them in the some of the configs it's probably bcoz they were not run in the CI and weren't caught.

Reply:
for tests, we use temp created configs, but It should be caught in tutorials